cs251tk/cli/cs251tk.py

```diff
@@ -70,8 +70,8 @@ def main():
                 results.append(result)
                 records.extend(recording)
 
-    table = tabulate(results, sort_by=args['sort'], partials=args['partials'])
     if not args['quiet']:
+        table = tabulate(results, sort_by=args['sort'], partials=args['partials'])
         print('\n' + table)
 
     if args['gist']:
```

Comment:
Could you duplicate the call to `table` inside of this `if`, please? It'll crash if the --gist argument is passed, without it.